api/app/applicationModel/mixins.py

```diff
@@ -0,0 +1,8 @@
+from flask_restful import reqparse
+
+
+class ApplicationFormMixin(object):
```

Comment:
What's the purpose of this mixin, since the functionality seems to already exist in the get method of ApplicationFromAPI ?